docs/metrics/customizing-the-sdk/README.md

```diff
@@ -31,6 +31,16 @@ using OpenTelemetry.Metrics;
 using var meterProvider = Sdk.CreateMeterProviderBuilder().Build();
 ```
 
+In a typical application, a single `MeterProvider` is created at application
+startup and disposed at application shutdown. It is important to ensure that the
+provider is not disposed too early. Actual mechanism depends on the application
+type. For example, in a typical ASP.NET application, `MeterProvider` is created
+in `Application_Start`, and disposed in `Application_End` (both methods part of
+Global.asp.cs file) as shown [here](../../../examples/AspNet/Global.asax.cs). In
+a typical ASP.NET Core application, `MeterProvider` lifetime is managed by
+leveraging the built-in Dependency Injection container as shown
+[here](../../../examples/AspNetCore/Startup.cs).
```

Comment:
These examples don't show `MeterProivder` setup.

Reply:
😃 I know. But in future we'll add. It at least shows "tracerprovider". so people might get inspired from its usage.